### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by stevelr**
Signed-off-by: stevelr <steve@cosmonic.com>

### CODE REVIEW

**Comment by autodidaddict** [comment]
Looks like nearly all the checks are failing, many with failures to find path-based dependencies:

```
Caused by:
  failed to load source for dependency `wasmbus-rpc`
```

File: nats/Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ chrono = "0.4"
 crossbeam="0.8"
 futures = "0.3"
 log = "0.4"
-nats = "0.13"
+nats = { version = "0.16.101", package = "nats-aflowt" }
```
Comment (brooksmtownsend) [inline nats/Cargo.toml:14]: What's the difference between this and just using `nats-aflowt 0.16.103`?

**Review by brooksmtownsend — APPROVED** [review]

File: nats/Cargo.toml (hunk 1 above)
Comment (stevelr) [reply nats/Cargo.toml:14]: it uses nats_aflowt but it's called nats inside the rust code.

Comment (stevelr) [reply nats/Cargo.toml:14]: there's no need to do this anymore, I can back it out. This was before I decided to standardize on aflowt being availble as `wasmbus_rpc::anats`, which also avoids a direct dependency

**Pull request merged**